app/functions.php

```diff
@@ -8,6 +8,53 @@
 *
 */
 
+/* ###### Get all posts ###### */
+
+function get_posts(){
+	
+	// Get Firebase JSON
+	$json = file_get_contents(FIREBASE_URL.'/posts.json');
```

Comment:
You mentioned before we could query with one param? I guess if we have either the type of status it would be worth doing this? Not for this PR obvs